BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -254,6 +261,22 @@ public class AppProcessor {
             services.errorReporter.log(error: error)
         }
     }
+
+    /// Starts timer to send organization events regularly
+    private func startEventTimer() {
+        sendEventTimer = Timer.scheduledTimer(withTimeInterval: 5, repeats: true) { _ in
+            Task {
+                await self.services.eventService.upload()
+            }
+        }
+        sendEventTimer?.tolerance = 10
+    }
+
+    /// Stops the timer for organization events
+    private func stopEventTimer() {
+        sendEventTimer?.fire()
```

Comment:
🤔 [In MAUI](https://github.com/bitwarden/mobile/blob/main/src/App/Platforms/iOS/AppDelegate.cs#L479) we're creating a [background task](https://developer.apple.com/documentation/uikit/uiapplication/1623031-beginbackgroundtask) when about to go to background so the events upload can have a bit more time before the app is suspended. Not sure if the scope changed here but wanted to raise this just in case.

Reply:
Ah, that makes sense! I'll make that change